File: service/SurfWebDefault/Model/Dtos/Players/PlayerStageDto.cs
namespace Model.Dtos.Players;

public class PlayerStageDto
{
    /// <summary>
    ///     阶段
    /// </summary>
    public int? Stage { get; set; }

    /// <summary>
    ///     时间
    /// </summary>
    public float Time { get; set; }

    /// <summary>
    ///     和wr差距
    /// </summary>
    public float GapTime { get; set; }

    /// <summary>
    ///     日期
    /// </summary>
    public DateTime Date { get; set; }
}

File: service/Java/SurfWeb/SurfWeb/system/src/main/java/com/surfweb/system/dto/players/PlayerStageDto.java
package com.surfweb.system.dto.players;

import lombok.Data;

import java.util.Date;

@Data
public class PlayerStageDto {

    /**
     * 阶段
     */
    private Integer stage;

    /**
     * 时间
     */
    private float time;

    /**
     * 和wr差距
     */
    private float gapTime;

    /**
     * 日期
     */
    private Date date;
}
